tests/functional/includes/html.event.polyfill.js

```diff
@@ -0,0 +1,9 @@
+// Needed for PhantomJS < 2.0
+
+if (!window.Event || typeof Event !== 'function') {
```

Comment:
Just checked with Juan, we will not be able to make this polyfill available via the polyfill service for a while. Also, this polyfill is temporary since Phantom 2 (soon available) will not require it.